### PR TITLE
use cim documents

### PR DESCRIPTION
**Pull request by cmgrd**
Create CIM XML documents for Accept/Reject.

Still missing Errors and conversion hereof. This will be addressed in another PR.

### CODE REVIEW

**Review by thbaan — APPROVED** [review]

**Review by thbaan — APPROVED** [review]

**Review by thbaan — APPROVED** [review]

**Review by erduna — APPROVED** [review]

**Review by kft — APPROVED** [review]

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDI/Acknowledgements/AcknowledgementXmlSerializer.cs [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                    xmlNamespace + message.DocumentName,
+                    new XAttribute(XNamespace.Xmlns + "cim", xmlNamespace),
+                    new XElement(xmlNamespace + "mRID", Guid.NewGuid().ToString()),
+                    new XElement(xmlNamespace + "type", message.Type),
+                    new XElement(xmlNamespace + "process.processType", message.ProcessType),
+                    new XElement(xmlNamespace + "sender_MarketParticipant.mRID", new XAttribute("codingScheme", message.Sender.CodingScheme), message.Sender.Id),
+                    new XElement(xmlNamespace + "sender_MarketParticipant.marketRole.type", message.Sender.Role),
+                    new XElement(xmlNamespace + "receiver_MarketParticipant.mRID", new XAttribute("codingScheme", message.Receiver.CodingScheme), message.Receiver.Id),
+                    new XElement(xmlNamespace + "receiver_MarketParticipant.marketRole.type", message.Receiver.Role),
+                    new XElement(xmlNamespace + "createdDateTime", message.CreatedDateTime),
```
Comment (erduna) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDI/Acknowledgements/AcknowledgementXmlSerializer.cs:37]: missing businessSector.type

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDI/Acknowledgements/AcknowledgementXmlSerializer.cs [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        {
+            var document = new XDocument(
+                new XElement(
+                    xmlNamespace + message.DocumentName,
+                    new XAttribute(XNamespace.Xmlns + "cim", xmlNamespace),
+                    new XElement(xmlNamespace + "mRID", Guid.NewGuid().ToString()),
+                    new XElement(xmlNamespace + "type", message.Type),
+                    new XElement(xmlNamespace + "process.processType", message.ProcessType),
+                    new XElement(xmlNamespace + "sender_MarketParticipant.mRID", new XAttribute("codingScheme", message.Sender.CodingScheme), message.Sender.Id),
+                    new XElement(xmlNamespace + "sender_MarketParticipant.marketRole.type", message.Sender.Role),
```
Comment (erduna) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDI/Acknowledgements/AcknowledgementXmlSerializer.cs:60]: missing businessSector.type

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDI/Acknowledgements/ConfirmMessage.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+namespace Energinet.DataHub.MarketRoles.Infrastructure.EDI.Acknowledgements
+{
+    public record ConfirmMessage(
+        string DocumentName,
+        string Type,
+        string ProcessType,
+        MarketParticipant Sender,
+        MarketParticipant Receiver,
+        Instant CreatedDateTime,
```
Comment (erduna) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDI/Acknowledgements/ConfirmMessage.cs:25]: missing missing businessSector.type

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDI/Acknowledgements/RejectMessage.cs [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+using NodaTime;
+
+namespace Energinet.DataHub.MarketRoles.Infrastructure.EDI.Acknowledgements
+{
+    public record RejectMessage(
+        string DocumentName,
+        string Type,
+        string ProcessType,
+        MarketParticipant Sender,
+        MarketParticipant Receiver,
```
Comment (erduna) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDI/Acknowledgements/RejectMessage.cs:24]: missing businessSector.type

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDI/MoveIn/RequestMoveInResultHandler.cs [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+                ProcessType: "E65",
+                Sender: new MarketParticipant(
+                    Id: "DataHub GLN", // TODO: Use correct GLN
+                    CodingScheme: "9",
+                    Role: "EZ"),
+                Receiver: new MarketParticipant(
+                    Id: request.EnergySupplierGlnNumber,
+                    CodingScheme: "9",
+                    Role: "DDQ"),
+                CreatedDateTime: _dateTimeProvider.Now(),
```
Comment (erduna) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDI/MoveIn/RequestMoveInResultHandler.cs:64]: missing businessSector.type

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDI/MoveIn/RequestMoveInResultHandler.cs [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+                ReasonCode: "39",
```
Comment (erduna) [inline source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDI/MoveIn/RequestMoveInResultHandler.cs:102]: missing businessSector.type

**Review by erduna — CHANGES_REQUESTED** [review]

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDI/Acknowledgements/AcknowledgementXmlSerializer.cs (hunk 1 above)
Comment (cmgrd) [reply source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDI/Acknowledgements/AcknowledgementXmlSerializer.cs:38]: Added.

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDI/Acknowledgements/AcknowledgementXmlSerializer.cs (hunk 2 above)
Comment (cmgrd) [reply source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDI/Acknowledgements/AcknowledgementXmlSerializer.cs:62]: Added.

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDI/Acknowledgements/ConfirmMessage.cs (hunk 3 above)
Comment (cmgrd) [reply source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDI/Acknowledgements/ConfirmMessage.cs:27]: Added.

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDI/Acknowledgements/RejectMessage.cs (hunk 4 above)
Comment (cmgrd) [reply source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDI/Acknowledgements/RejectMessage.cs:26]: Added.

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDI/MoveIn/RequestMoveInResultHandler.cs (hunk 5 above)
Comment (cmgrd) [reply source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDI/MoveIn/RequestMoveInResultHandler.cs:66]: Added.

File: source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDI/MoveIn/RequestMoveInResultHandler.cs (hunk 6 above)
Comment (cmgrd) [reply source/business-workflow/source/Energinet.DataHub.MarketRoles.Infrastructure/EDI/MoveIn/RequestMoveInResultHandler.cs:106]: Added.

**Review by erduna — APPROVED** [review]

**Review by Korgath — APPROVED** [review]

**Pull request merged**